src/components/fields/edit/LatLongEditor.jsx

```diff
@@ -46,7 +76,13 @@ export default function LatLongEditor({
           value={currentLatitudeString}
           onChange={e => {
             const inputValue = e.target.value;
-            onChange([parseFloat(inputValue), currentLongitude]);
+            const floatValue = parseFloat(inputValue);
+            if (Number.isNaN(floatValue)) {
+              onChange([parseFloat(inputValue), currentLongitude]);
+            } else {
+              onChange([null, currentLongitude]);
```

Comment:
Looks like this line and the line below it could possibly be a mix up between latitude and longitude?

Reply:
it's correct as-is actually! since the latitude is changing, onChange is recording the new latitude value plus the current (unchanged) longitude value